homeassistant/components/media_player/denonavr.py

```diff
@@ -203,7 +203,9 @@ def media_image_url(self):
     @property
     def media_title(self):
         """Title of current playing media."""
-        if self._title is not None:
+        if self._current_source not in self._receiver._playing_func_list:
```

Comment:
you can push to version 0.3.1 of denonavr to use **self._receiver.playing_func_list** at this place.